docs/whats-new.md

```diff
@@ -3,28 +3,60 @@
 In addition to these notes, always check the [Upgrade Guide](/#/documentation/upgrade-guide) when considering adopting a new release.
 
 
+
 ## Version 5.2
 
 **IN DEVELOPMENT**
 
 Date: Target April, 2018
 
-## Transform class (New, WebGL2)
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/webgl2.jpg" />
+        <p><i>New Transform Class</i></p>
+      </td>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/webgl2.jpg" />
+        <p><i>Example 14</i></p>
+      </td>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/webgl2.jpg" />
+        <p><i>Example 15</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+## Transform class (New, WebGL2) (Experimental)
+
+The new experimental [`Transform`](/#/documentation/api-reference/transform) class provides an easy-to-use interface to perform Transform Feedback operations.
+
+
+## Framebuffer Class
+
+** "Asynchronous" Pixel Readback** (WebGL2) - A new method [`readPixelsToBuffer`](/#/documentation/api-reference/framebuffer) is added to asynchronously read pixel data into a `Buffer` object. This way applications have the freedom to reduce the CPU-GPU sync time or completely avoid it by using the `Buffer` object as data source for the GPU.
+
 
-The new [`Transform`](/#/documentation/api-reference/transform) class provides an easy-to-use interface to perform Transform Feedback operations.
+## Easier to Run luma.gl under Node.js
 
-## Framebuffer Asynchronous Read (New, WebGL2)
+When running under Node.js, luma.gl now automatically loads headless-gl (`gl`) if installed on the system. It is no longer required to `import "luma.gl/headless"`. See [Using with Node](/#/documentation/get-started/using-with-node) and the Upgrade Guide.
 
-A new method [`readPixelsToBuffer`](/#/documentation/api-reference/framebuffer) is added to asynchronously read pixel data into a `Buffer` object. This way applications have the freedom to reduce the CPU-GPU sync time or completely avoid it by using the `Buffer` object as data source for the GPU.
 
-## Node.js Support Changes
+## Dist Size Reduction
 
-When running under Node.js, luma.gl will now automatically attempt to load `gl` (headless-gl) if installed on your system. It is no longer required to `import "luma.gl/headless"`. This should somewhat simplify running luma.gl code under Node.js. See [Using with Node](/#/documentation/get-started/using-with-node) and the Upgrade Guide.
+Bundle sizes for a minimal luma.gl app with webpack 2.
 
+| Dist | 5.1.4 Bundle (Compressed) | 5.2.0 Bundle (Compressed) | Comments |
```

Comment:
@Pessimistress I feel that this section is to detailed for "what's new". I'd like to talk about numbers and techniques for bundle size reduction somewhere in the docs, and link to that here. Do you have any suggestions where to put it?